db/migrations/20230614145615_add_size_to_volume.sql

```diff
@@ -0,0 +1,10 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE unweave.volume ADD COLUMN size integer NOT NULL DEFAULT 0;
+
+-- +goose StatementEnd
+
+-- +goose Down
+-- +goose StatementBegin
+select 'down SQL query';
+-- +goose StatementEnd
```

Comment:
the size column was missing from the db, without it, how do we do billing? this migration will help us update billing